integrations/whatsapp/src/tracking.ts

```diff
@@ -0,0 +1,77 @@
+import { Analytics } from '@segment/analytics-node'
+import integrationDefinition from 'integration.definition'
+import * as bp from '.botpress'
+
+// safely initialize analytics instance
+let analytics: Analytics | undefined
+try {
+  if (!bp.secrets.SEGMENT_KEY) {
+    throw new Error('Missing Segment key')
+  }
+  analytics = new Analytics({ writeKey: bp.secrets.SEGMENT_KEY, flushAt: 1, httpRequestTimeout: 2000 })
+} catch (error) {
+  console.error('Could not initialize Segment analytics instance.')
```

Comment:
Just realized we'd lose the original error here, so best to include it in the logs when it happens 😄 

```suggestion
  console.error('Could not initialize Segment analytics instance.', error)
```